src/main/scala/scala/tools/refactoring/implementations/oimports/Region.scala

```diff
@@ -69,7 +69,7 @@ object Region {
 
   private def cutPrefixSuffix(imp: Global#Import, source: SourceFile): (String, List[String]) = {
     val printedImport = source.content.slice(imp.pos.start, imp.pos.end).mkString
-    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*(\w|\d|_|-)+(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
+    val prefixPatternWithCommentInside = """import (((\/\*.*\*\/)*((\w|\d|_|-)+|(\`.*\`)+)(\/\*.*\*\/)*)\.)+(\/\*.*\*\/)*""".r
```

Comment:
@mlangc you tried to break `splitUntilTypeParameterBracket(..)` but what about `fakeSelectTreeFromType(..)`? General problem is that this code is really buggy. Sorry I followed this path.
about extra space `\s+` isn't good enough? anyway I try `SourceWithMarker`. Previously I didn't because there wasn't option of reversed walk. Now it is done.